cmake/tribits/ctest_driver/TribitsCTestDriverCore.cmake

```diff
@@ -1004,31 +1004,46 @@ INCLUDE(${CMAKE_CURRENT_LIST_DIR}/TribitsCTestDriverCoreHelpers.cmake)
 # and other results and submitted and displayed on CDash (but not what CDash
 # site(s) or project(s) they are submitted to).  These options can all be set
 # in the CTest -S script using ``SET()`` statements before
-# ``TRIBITS_CTEST_DRIVER()`` is called and can be overridden in the env.
+# ``TRIBITS_CTEST_DRIVER()`` is called and can be overridden in the env when
+# running the CTest -S driver script.
 #
 #   .. _CTEST_TEST_TYPE:
 #
 #   ``CTEST_TEST_TYPE=[Nightly|Continuous|Experimental]``
 #
-#     Determines the type of build.  This value is passed in as the first
+#     Determines the model for build.  This value is passed in as the first
 #     argument to the built-in CTest function ``CTEST_START()``.  Valid values
-#     include ``Nightly``, ``Continuous``, and ``Experimental``.  This also
-#     defines the default value for `${PROJECT_NAME}_TESTING_TRACK`_ as well
-#     as defines the default value for
+#     include ``Nightly``, ``Continuous``, and ``Experimental``.  As far a
```

Comment:
```suggestion
#     include ``Nightly``, ``Continuous``, and ``Experimental``.  As far as
```

Reply:
Addressed in commit https://github.com/bartlettroscoe/TriBITS/commit/c38b89ca8c836b4f9d9ded010b9825aa959b39e0